lib/wallaroo/core/autoscale/autoscale.pony

```diff
@@ -583,6 +633,64 @@ actor Autoscale
   // COORDINATOR
   //////////////////////////////////
 
+  be try_shrink(local_topology: LocalTopologyInitializer,
+    target_workers: Array[WorkerName] val, shrink_count: U64,
+    response_fn: TryShrinkResponseFn)
+  =>
+    if (_worker_name == _primary_worker) then
+      _phase.try_shrink(local_topology, target_workers, shrink_count,
+        response_fn)
+    else
+      _waiting_connections.insert(
+        {(abs) =>
```

Comment:
I'm not sure what `abs` means

Reply:
ah, array bytes.  I think because of the distance it would be worth using a descriptive name